tools/deployment/package-helm/templates/otel-collector-deployment.yaml

```
@@ -0,0 +1,39 @@
+    matchLabels:
+      {{- include "clp.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: "otel-collector"
+  template:
+    metadata:
+      labels:
+        {{- include "clp.labels" . | nindent 8 }}
+        app.kubernetes.io/component: "otel-collector"
+    spec:
+      containers:
+        - name: otel-collector
+          image: {{ .Values.clpConfig.telemetry.collector.image | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)